### PR TITLE
:camping: Align cloud benchmark

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

Make cloud benchmark namespace aware. All commands go to a specific namespace, namespace is not switched anymore.

Use OS dependent sed function.

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/0.25`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

File: benchmarks/setup/cloud-default/Makefile [hunk 1]
```diff
@@ -1,45 +1,47 @@
+namespace ?= default
```
Comment (korthout) [inline benchmarks/setup/cloud-default/Makefile:1]: ❓ Why a safe assignment and not a regular assignment (i.e. `=`)? I'm not that familiar with the assignment syntax of Makefiles, and I couldn't find a clear answer on why you'd want to do this. As I understand it, you'll replace the `default` literal anyways with `sed`.

**Review by korthout — APPROVED** [review]
LGTM ⛺  Just 1 question about syntax

File: benchmarks/setup/cloud-default/Makefile (hunk 1 above)
Comment (Zelldon) [reply benchmarks/setup/cloud-default/Makefile:1]: One reason is that I just copied it from the other Makefile https://github.com/camunda-cloud/zeebe/blob/develop/benchmarks/setup/default/Makefile#L1

The other reason is I think to set the variable also from outside, like https://stackoverflow.com/questions/2826029/passing-additional-variables-from-command-line-to-make

Comment (korthout) [reply benchmarks/setup/cloud-default/Makefile:1]: Thanks 👍  I learned something today 🎉

**Pull request merged**